mage_ai/server/app.py

```diff
@@ -335,34 +336,42 @@ def update_pipeline(id):
 #     return feature_set.column(column_name)
 
 
-def clean_df(df, name):
+def clean_df(df, name, verbose=False):
     feature_set = FeatureSet(df=df, name=name)
 
-    result = clean_data(df)
+    result = clean_data(df, verbose=verbose)
 
     feature_set.write_files(result)
     return (feature_set, result['df'])
 
 
-def clean_df_with_pipeline(df, id=None, path=None, remote_id=None, mage_api_key=None):
+def clean_df_with_pipeline(
+    df, id=None, path=None, remote_id=None, mage_api_key=None, verbose=False
+):
     pipeline = None
     if id is not None:
-        pipeline = Pipeline(id=id).pipeline
+        with VerboseFunctionExec(f'Loading pipeline with uuid \'{id}\' from file', verbose=verbose):
+            pipeline = Pipeline(id=id).pipeline
     elif path is not None:
-        pipeline = Pipeline(path=path).pipeline
+        with VerboseFunctionExec(f'Loading pipeline from path \'{path}\'', verbose=verbose):
+            pipeline = Pipeline(path=path).pipeline
     elif remote_id is not None:
-        final_key = mage_api_key if mage_api_key is not None else api_key
-        pipeline = BasePipeline(actions=Mage().get_pipeline_actions(remote_id, final_key))
+        with VerboseFunctionExec(
+            f'Loading pipeline \'{remote_id}\' from Mage servers', verbose=verbose
+        ):
+            final_key = mage_api_key if mage_api_key is not None else api_key
+            pipeline = BasePipeline(actions=Mage().get_pipeline_actions(remote_id, final_key))
```

Comment:
Did we pass verbose parameter to pipeline?

Reply:
It is added below on line 367